### PR TITLE
Vboulos acm210 ocp4.16

### PR DESCRIPTION
**Pull request by vboulos**
- Create OPP new job to run OCP 4.16 with ACM 2.10
- Update firewatch priorities to fix the cascading failures from being reported
- Update the component name in firewatch to become "OPPQE" instead of just "QE"

### CODE REVIEW

**Review by gparvin — APPROVED** [review]

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

File: ci-operator/jobs/stolostron/policy-collection/stolostron-policy-collection-main-periodics.yaml [hunk 1]
```diff
@@ -106,22 +106,11 @@ periodics:
...
-    slack:
-      channel: '#opp-discussion'
-      job_states_to_report:
-      - success
-      - failure
-      - error
-      report_template: '{{if eq .Status.State "success"}} :slack-green: Job *{{.Spec.Job}}*
-        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :failed:
-        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
-        logs> {{end}}'
```
Comment (gparvin) [inline ci-operator/jobs/stolostron/policy-collection/stolostron-policy-collection-main-periodics.yaml:124]: Is slack integration removal on purpose?

Comment (vboulos) [reply ci-operator/jobs/stolostron/policy-collection/stolostron-policy-collection-main-periodics.yaml]: I added it back as it looks like it was removed after I ran the "make update" command

**Review by gparvin — APPROVED** [review]

**Pull request merged**